app/src/main/java/com/lightricks/feedexercise/data/Models.kt

```diff
@@ -8,10 +8,9 @@ import kotlinx.serialization.Serializable
  * In a real-life application this is template meta-data for a user's project.
  * The rest of the properties are left out for brevity.
  */
-@Serializable
 data class FeedItem(
-    val id: String,
-    @SerialName("templateThumbnailURI")
+    val id: Int = 0,
+    val feedItemId: String,
```

Comment:
I am not sure I understand. Why do we need two ids and of two different types?

Reply:
Since the webservice is the producer of the data so I don't see a reason why not to rely on it.